README.md

```diff
@@ -1 +1,32 @@
-# mongodb-operator
\ No newline at end of file
+# Charmed MongoDB Operator
+
+
+## Description
+
+The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database.
+
+MongoDB is a general purpose distributed document database. This operator charm deploys and operates MongoDB on physical or virtual machines as a single replica.
+
+
+## Usage
+
+Until the MongoDB Machine Charm is published, you need to follow the build & deploy instructions from [CONTRIBUTING.md](./CONTRIBUTING.md) to deploy the charm.
+
+
+## Relations
+
+Supported [relations](https://juju.is/docs/olm/relations):
+
+- This charm supports peer relations, which provides replica sets for the database.
+
+The charm will implement both [sharding](https://docs.mongodb.com/manual/sharding/) and [replica set](https://docs.mongodb.com/manual/replication/). To do this, it will need an extra component, the `config-server`.
+
+Future relations will include:
+
+- Mongo router `mongos` charm deployed alongside MongoDB charm. It will handle a sharded deployment, and it can work as a MongoDB database interface.
+- A MongoDB database `shard` interface. It will be used by the mongos charm as well.
+- A `config-server` interface used by the mongos charm.
+
+## Contributing
```

Comment:
```suggestion
## License
The Charmed MongoDB Operator is free software, distributed under the Apache Software License, version 2.0. See [LICENSE](https://github.com/MiaAltieri/mongodb-operator/master/blob/LICENSE) for more information.

## Security
Security issues in the Charmed MongoDB Operator can be reported through [LaunchPad](https://wiki.ubuntu.com/DebuggingSecurity#How%20to%20File). Please do not file GitHub issues about security issues.

## Contributing
```

Reply:
I'm guessing this needs to be added to the template as well, since it's for all charms.